estate/models/estate_property.py

```diff
@@ -0,0 +1,112 @@
+from dateutil.relativedelta import relativedelta
+from odoo import api, fields, models
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate property"
+
+    name = fields.Char(string="Title", required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(string="Available From", default=fields.Date.today() + relativedelta(months=3), copy=False)
```

Comment:
```suggestion
    date_availability = fields.Date(string="Available From", default=lambda self: fields.Date.context_today(self) + relativedelta(months=3), copy=False)
```


`date_availability`'s default was calculated only once at server launch. Wrapping it in a `lambda` fixes this by computing it dynamically for each new record. I also used `fields.Date.context_today(self)` for timezone correctness.